src/extensionVariables.ts

```diff
@@ -18,6 +18,7 @@ export namespace ext {
     export let outputChannel: OutputChannel;
     export let reporter: TelemetryReporter | undefined;
     export let tree: AzureTreeDataProvider;
+    export let cosmosView: TreeView<AzureTreeItem<ISubscriptionRoot>>;
```

Comment:
I prefer to just name this `treeView`. This whole repo is Cosmos, so there's no need to have that in the name